Tasks/KubectlInstallerV0/src/utils.ts

```diff
@@ -0,0 +1,28 @@
+"use strict"
+
+import tl = require("azure-pipelines-task-lib/task");
+import toolLib = require("azure-pipelines-tool-lib/tool");
+import kubectlutility = require("utility-common/kubectlutility");
+
+export async function getKuberctlVersion(): Promise<string> {
+    let kubectlVersion = tl.getInput("kubectlVersion");
+    if(kubectlVersion && kubectlVersion != "latest") {
+        return sanitizeVersionString(kubectlVersion.trim());
+    }
+
+    return await kubectlutility.getStableKubectlVersion();
+}
+
+export async function downloadKubectl(version : string): Promise<string> {
+    return await kubectlutility.downloadKubectl(version);
+}
+
+// handle user input scenerios
+export function sanitizeVersionString(inputVersion: string) : string{
+    var version = toolLib.cleanVersion(inputVersion);
+    if(!version) {
+        throw new Error(tl.loc("NotAValidSemverVersion"));
+    }
+    
+    return "v"+version;
```

Comment:
should not we check whether it starts with v or not and based on that either add it or not.

Reply:
the above line "toollib.cleanVersion"  removes the v prefix

![image](https://user-images.githubusercontent.com/8982085/55327167-1af44b00-54a7-11e9-8f73-608708467077.png)